Assessment.md

```diff
@@ -27,20 +27,25 @@ class MainActivity : ComponentActivity(), SMKitUIWorkoutListener {
 fun startAssessment() {
     try {
         // In order to start Sency's blueprint assessment
-        smKitUI.startAssessment(listener = this)
+        smKitUI.startAssessment(
+            Fitness, // Assessment Type
+            listener = this, // Assessment Listener
+            userData = UserData(age = 28, gender = Gender.Female) // User Data 
```

Comment:
@SencyCaspit we should mention that it's optional

Reply:
@ofergoldstein i mentioned it below, you want me to mention it also up here?

Reply:
@SencyCaspit yes
`userData = UserData(age = 28, gender = Gender.Female) // User Data (optional) `